util/parsec_project.py

```
@@ -0,0 +1,241 @@
+
+        Returns:
+            dict[str, Any]: The result of running Parsec at the given path.
+
+        Raises:
+            FileNotFoundError: If the given path is invalid
+            RuntimeError: If an error occurs while running Parsec, or if
+                Parsec fails to produce an analysis.json file.
+        """
+        if path.is_file():
+            cmd = ["parsec", "--rename-main=false", "--add-instr=false", str(path)]
+            result = subprocess.run(cmd, capture_output=True, text=True)
```

Ruff (0.15.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)